packages/rulesets/src/spectral/test/utils.ts

```diff
@@ -14,12 +14,16 @@ export function buildLinter(rulesets: any[], testedRuleName: string, useNoopReso
     })
   })
 
-  // Delete the "stagingOnly" property for the same reasons as we do so in
+  // Delete "stagingOnly", "rpcGuidelineCode" property for the same reasons as we do so in
```

Comment:
May as well copy it here

```suggestion
  // Delete "stagingOnly", "rpcGuidelineCode" properties as constructing @stoplight/spectral-core Ruleset downstream would fail validation .
```

Reply:
This is for test, we dont send the req. to spectral, its just for internal execution